docs/getting-started.stories.mdx

```diff
@@ -28,7 +28,7 @@ npm install carbon-react
 You will need to install the following dependencies in your project, these are peer-dependencies of carbon-react and are required.
 
 ```sh
-npm install react@^16.12.0 react-dom@^16.12.0 i18n-js@^3.0.0 styled-components@^4.4.1
+npm install react@^16.12.0 react-dom@^16.12.0 i18n-js@^3.0.0 styled-components@^4.4.1 draft-js@^0.11.5
```

Comment:
So any consumer of carbon-react needs to install `draft-js` even if they're not using `TextEditor` or `Note`?

Reply:
Yeah good point, definitely misleading having it added there. I've updated it